layouts/error.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <div class="layout-default">
+      <div class="pt-4 mt-12">
+        <center>
```

Comment:
@CryptoBender21 
I think we should not use the `<center>` tag as it's not supported in HTML5.
The VSCode itself complains it I guess.

Reply:
Agreed, was taken from a different template for the sake of consistency